src/Models/Model.php

```diff
@@ -74,8 +79,8 @@ public function getFillable()
         // Use the list of translatable attributes on our base model
         if (
             blank($fillable) &&
-            Str::contains($class = get_class($this), 'Models\Translations') &&
-            property_exists($class, 'baseModuleModel')
+            $this->isTranslationModel() &&
+            property_exists($this, 'baseModuleModel')
```

Comment:
If we used an interface for translatable models, we wouldn't need to do these sorts of checks.